components/atom/input/README.md

```diff
@@ -0,0 +1,125 @@
+# AtomInput
+
+> Form input
+
+<!-- ![](./assets/preview.png) -->
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-atom-input --save
+```
+
+## Types
+
+### **Text**
+```js
+import AtomInput, {atomInputTypes} from '@s-ui/react-atom-input'
+
+return (
+    <AtomInput
+      label='Text input'
+      name='text'
+      type={atomInputTypes.text}
+    />
+)
+```
+
+### **Number**
+```js
+import AtomInput, {atomInputTypes} from '@s-ui/react-atom-input'
+
+return (
+    <AtomInput
+      label='Number input'
+      type={atomInputTypes.number}
+    />
+)
+```
+
+### **Date**
+```js
+import AtomInput, {atomInputTypes} from '@s-ui/react-atom-input'
+
+return (
+    <AtomInput
+      label='Date input'
+      type={atomInputTypes.date}
+    />
+)
+```
+
+### **Mask**
+Is used in case the input should match a regex, e.g postal code or bank account. See [imaskjs](https://unmanner.github.io/imaskjs/guide.html#common) masks for more info
+
```

Comment:
Mask → no clear documentation? Typical use cases?
shouldn't it accept native regular expressions?